### PR TITLE
add back TestGaiaApp_Export 

### PR DESCRIPTION
**Pull request by okwme**
Closes items in #1909  related to
 - TestGaiaApp_Export
 - Investigate the `app/helpers/test_helpers.go` setup function that used to pre-populate with an account and a validator.

### CODE REVIEW

File: app/helpers/test_helpers.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/tendermint/tendermint/crypto"
+
 	// "github.com/tendermint/tendermint/crypto/secp256k1"
```
Comment (mmulji-ic) [inline app/helpers/test_helpers.go]: Can we take this out?

Comment (okwme) [reply app/helpers/test_helpers.go]: Nice catch. Removed

File: app/app_test.go [hunk 2]
```diff
@@ -5,11 +5,11 @@ import (
 
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	gaia "github.com/cosmos/gaia/v8/app"
```
Comment (glnro) [inline app/app_test.go:8]: why move these up here?

**Review by glnro — APPROVED** [review]
Looks fine so far, in the future we should try to make this as consistent as possible with the [sdk](https://github.com/cosmos/cosmos-sdk/blob/c0d85e30c35921aa0104091f9c45517cd01315ae/simapp/test_helpers.go#L92)...especially when we eventually play #1883

File: app/helpers/test_helpers.go [hunk 3]
```diff
@@ -161,11 +140,7 @@ func setup(withGenesis bool, invCheckPeriod uint) (*gaiaapp.GaiaApp, gaiaapp.Gen
 		encCdc,
 		EmptyAppOptions{},
 	)
-	if withGenesis {
-		return gaiaApp, gaiaapp.NewDefaultGenesisState()
-	}
-
-	return gaiaApp, gaiaapp.GenesisState{}
+	return gaiaApp, gaiaapp.NewDefaultGenesisState()
```
Comment (mmulji-ic) [inline app/helpers/test_helpers.go:143]: We use NewDefaultGenesisState, so MakeTestEncodingConfig is setup ... what does this give us?

**Pull request merged**